src/test/java/org/gridsuite/shortcircuit/server/repositories/FaultResultRepositoryTest.java

```diff
@@ -131,6 +141,38 @@ void faultResultFilterWithPageableTest(ShortCircuitAnalysisResultEntity resultEn
         assertFaultEqualsInOrder(faultPage, Comparator.comparing(FaultResultEntity::getNbLimitViolations).thenComparing(o -> o.getFaultResultUuid().toString()));
     }
 
+    @ParameterizedTest(name = "[{index}] Using the filter(s) {1} should return the given entities")
+    @MethodSource({
+        "provideFeederFieldsStartFiltersSortedAsc",
+        "provideFeederFieldsContainFiltersSortedDesc",
+        "provideFeederFieldsUnfilteredSortedAscAndDesc"
```

Comment:
Ideally, a test should have only one reason to fail.
With parametized test it's really easy and quick to make lot of tests.
I think you should split them a bit more, filtering and sorting is two different functionnalities.

Reply:
I did separate the first two. But I kept one who mixes both. Given the problems I had to mix both, I think it might be useful to have one test about it.